src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProcessor.cs

```diff
@@ -50,21 +50,10 @@ public virtual void EnqueueMessage(LogMessageEntry message)
         }
 
         // for testing
-        internal virtual void WriteMessage(LogMessageEntry message)
+        internal virtual void WriteMessage(LogMessageEntry entry)
         {
-            IConsole console = message.LogAsError ? ErrorConsole : Console;
-
-            if (message.TimeStamp != null)
-            {
-                console.Write(message.TimeStamp, message.MessageColor, message.MessageColor);
-            }
-
-            if (message.LevelString != null)
-            {
-                console.Write(message.LevelString, message.LevelBackground, message.LevelForeground);
-            }
-
-            console.Write(message.Message, message.MessageColor, message.MessageColor);
+            IConsole console = entry.LogAsError ? ErrorConsole : Console;
+            console.Write(entry.Message);
```

Comment:
>Write [](start = 20, length = 5)

wouldn't think will loose logging the other information in the entry e.g. timestamp and level?

Reply:
I am seeing now we have changed the entry and removed the other information like the timestamp and the level. how we can log such missing information? 

---
In reply to: [448664759](https://github.com/dotnet/runtime/pull/38616#discussion_r448664759) [](ancestors = 448664759)

Reply:
entry.Message here already contains info such as timestamp and level and wrapped around whatever format dictated by the Formatter that wrote it into the TextWriter.